addons/policies/bootstrap_heka_policies.json

```diff
@@ -0,0 +1,35 @@
+{
+  "entities":
+  [
+    {
+    "typeName": "AuthPolicy",
+    "attributes":
+    {
+      "name": "DENY_DATA_ACCESS_GUEST",
+      "qualifiedName": "DENY_DATA_ACCESS_GUEST",
+      "description": "deny data access for guest users",
+      "policyCategory": "bootstrap",
+      "policySubCategory": "data",
+      "policyServiceName": "heka",
+      "policyType": "deny",
+      "policyPriority": 0,
```

Comment:
BTW 0 represents low priority, I think we want this to have higher priority, change it to "1" to specify higher priority

Reply:
Right, it should get first priority before anything else